controllers/configuration_controller.go

```diff
@@ -1110,6 +1120,27 @@ type TFState struct {
 	Outputs map[string]TfStateProperty `json:"outputs"`
 }
 
+func (meta *TFConfigurationMeta) isTFStateGenerated(ctx context.Context) bool {
+	// 1. exist backend
+	if meta.Backend == nil {
+		return false
+	}
+	// 2. and exist tfstate file
+	tfStateJSON, err := meta.Backend.GetTFStateJSON(ctx)
+	if err != nil {
```

Comment:
if kerror.IsNotFound(err) {
  return false
}

Reply:
If the backend is S3, will there be any issues with kerrors.IsNotFound?